res/enkf/analysis_config.py

```diff
@@ -245,9 +243,8 @@ def activeModuleName(self):
         """:rtype: str"""
         return self._get_active_module_name()
 
-    def getModuleList(self) -> StringList:
-        """:rtype: StringList"""
-        return self._get_module_list()
+    def getModuleList(self):
```

Comment:
Can you add the type hint back?

Reply:
Can you also fix the places using this call? They are explicitly casting to list, that is no longer required.

Reply:
Thanks!